src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -607,6 +612,9 @@ public String toString() {
 		if (isNotBlank(getAlbum())) {
 			result.append(", Album: ").append(getAlbum());
 		}
+		if (isNotBlank(getAlbumArtist())) {
+			result.append(", Album Album: ").append(getAlbumArtist());
```

Comment:
Just a typo here
```suggestion
			result.append(", Album Artist: ").append(getAlbumArtist());
```

Reply:
Of course, no problem with that!

Reply:
I think you can choose to apply my suggestions from within the browser, since I don't have permission to do it :)